language/tools/mirai-dataflow-analysis/rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2021-08-17
```

Comment:
Why this is needed here?

Reply:
mirai-dataflow sets the nightly version of the crate being analyzed to the version that MIRAI uses. That version is read from this file.

Reply:
Does it mean that every time MIRAI update the version, this file needs to be sync-ed as well?

Reply:
Yes, it will need to be kept in sync.